Composer/packages/client/src/components/GetStarted/GetStartedLearn.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import formatMessage from 'format-message';
+import { Link } from 'office-ui-fabric-react/lib/Link';
+
+import TelemetryClient from '../../telemetry/TelemetryClient';
+
+import { h3Style, ulStyle, liStyle } from './styles';
+
+export const GetStartedLearn: React.FC = () => {
+  const linkToAdaptiveExpressions =
```

Comment:
Nit: Move all the consts out of the component function so you don't redeclare them every render.

Reply:
easy